libs/text-splitters/langchain_text_splitters/character.py

```diff
@@ -577,3 +577,62 @@ def get_separators_for_language(language: Language) -> List[str]:
                 f"Language {language} is not supported! "
                 f"Please choose from {list(Language)}"
             )
+
+
+class SemanticTiktokenTextSplitter(TextSplitter):
+    """Splitting text that looks at characters."""
+
+    def __init__(
+        self,
+        *,
+        max_characters: int = 200,
+        model_name: str = "gpt-3.5-turbo",
```

Comment:
It doesn't look like model_name should be assigned a default value. assigning a default value will make it such that users are more likely to end up with incorrect code because they're not aware there's a configuration parameter